skfem/utils.py

```diff
@@ -187,7 +174,7 @@ def callback(x):
         if verbose:
             print(np.linalg.norm(x))
 
-    def solver(A, b):
+    def solver(A, b, **kwargs):
```

Comment:
Yes, we do want to accept here keyword-arguments for passing on to the solver at solve-time but taking care not to smother those passed in here for defining the solver.  I'm thinking of cases like a heat equation for which the solver is used repeatedly with the same operator `A`, preconditioner `M`, &c.,  but changing right-hand side `b` and starting guess `x0`.

Reply:
See #307.

Reply:
Makes sense, let's go with this approach.